### ISSUE TITLE
Feature Request: Spanish/Language Toggles

### ISSUE THREAD

**Issue by sashperso** [openpracticelibrary/openpracticelibrary#1897]
I've noticed that there are some practices that have been translated into Spanish and available on the library (Tormenta de eventos/Event Storming and Meditacion/Meditation come to mind). It would be awesome if visitors would have the ability to toggle directly on either of these practices to see it in their preferred language. This could be helpful in the future if translations are done by other people in different languages!

**Comment by kattekrab** [comment]
Yes! Thanks @sashperso - I really want to tackle the multi-lingual thing.

**Comment by kattekrab** [comment]
Merci @Moosh-be - that would be amazing! 🙏 💖
